Makefile

```diff
@@ -457,3 +457,18 @@ repl-clojure: ##@repl Start Clojure repl for mobile App
 	yarn shadow-cljs cljs-repl mobile
 
 repl-nix: nix-repl ##@repl Start an interactive Nix REPL
+
+#--------------
+# Dev Automation Flows
+#--------------
+
+automate-login: export TARGET := default
+automate-login: ##@other runs flow for login or onboarding app on simulator/emulator
+	maestro test "maestro/create-account-or-login.yaml"
+
+automate-custom: export TARGET := default
+automate-custom: ##@other runs any custom maestro automation flow on simulator/emulator
```

Comment:
Maybe a shorter `auto-*` prefix would be easier to type.

Also are we sure this should go into `other` section of help? Maybe it should have it's own `auto` section.